staging/src/k8s.io/apiserver/pkg/storage/selection_predicate.go

```diff
@@ -127,6 +127,9 @@ func (s *SelectionPredicate) MatchesSingle() (string, bool) {
 
 // Empty returns true if the predicate performs no filtering.
 func (s *SelectionPredicate) Empty() bool {
+	if s.Label == nil && s.Field == nil {
```

Comment:
Why was this necessary? Shouldn't `LabelSelector.Empty()` already handle this?

(Did this leak in from some other PR?)